### PR TITLE
fix for #34

### PR DESCRIPTION
**Pull request by ayberk**
Please see line 116,118 and 129. Other changes due to my editor automatically removing trailing space.

### CODE REVIEW

**Comment by harsham05** [comment]
@ayberk Please provide comma fix for value-similarity.py

**Comment by ayberk** [comment]
@harsham05 will do, thanks for noticing!

File: similarity.py [hunk 1]
```diff
@@ -124,19 +125,18 @@ def main(argv = None):
 		# now compute the specific resemblance and containment scores
 		for filename in file_parsed_data:
 			overlap = {}
-			overlap = set(file_parsed_data[filename].keys()) & set(union_feature_names) 
-			resemblance_scores[filename] = float(len(overlap))/total_num_features
-
+			overlap = set(file_parsed_data[filename].keys()) & set(union_feature_names)
+			resemblance_scores[filename_stripped] = float(len(overlap))/total_num_features
```
Comment (harsham05) [inline similarity.py]: This is supposed to be resemblance_scores[filename] as it is the iterator variable

File: similarity.py [hunk 2]
```diff
@@ -94,15 +94,15 @@ def main(argv = None):
...
-			filename_list.append(filename)		
-
-		if len(filename_list) <2 :		
+		filenames = [x.strip() for x in filenames]
+		filenames = [filenames[k].strip('\'\n') for k in range(len(filenames))]
+		for filename in filenames :
+			if not os.path.isfile(os.path.join(dirFile, filename)):
+				continue
+			filename = os.path.join(dirFile, filename) if dirFile else filename
+			filename_list.append(filename)
```
Comment (harsham05) [inline similarity.py]: Can you please do the comma fix here?
**filename_list.append(filename.replace(",","")**

File: similarity.py (hunk 1 above)
Comment (ayberk) [inline similarity.py]: done.

File: similarity.py (hunk 2 above)
Comment (ayberk) [inline similarity.py]: I thought about it, but wouldn't it break line 115? I assumed `parsedData = parser.from_file(filename)` would open `filename` and return the parsed data.

Comment (harsham05) [inline similarity.py]: @ayberk Yes, you are right :)

**Comment by chrismattmann** [comment]
OK is this ready to merge @harsham05 @ayberk ?

**Comment by harsham05** [comment]
@chrismattmann yes :)

**Pull request merged**